db/migrate/20170410164012_remove_workspaces_collection_foreign_key.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveWorkspacesCollectionForeignKey < ActiveRecord::Migration[5.0]
+  def change
+    remove_foreign_key :workspaces, :collection
```

Comment:
i dug around a bit and i think we can keep the foreign key by changing its `on_delete` behavior with `:nullify` like so:

```
remove_foreign_key :workspaces, :collection
add_foreign_key :workspaces, :collection, on_delete: :nullify
```

As I understand it, we wouldn't need to change the model code at all.

See http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_foreign_key